### PR TITLE
client library - http get_model_status implementation

### PR DESCRIPTION
**Pull request by mkuczyns11**

### CODE REVIEW

**Review by mzegla — COMMENTED** [review]
Please make sure you run also style checks before merging

File: client/python/lib/ovmsclient/tfs_compat/http/serving_client.py [hunk 1]
```diff
@@ -107,7 +111,17 @@ def get_model_status(self, request):
             >>> type(response)
         '''
 
-        raise NotImplementedError
+        HttpClient._check_model_status_request(request)
+
+        raw_response = None
+        try:
+            raw_response = requests.get(f"http://{self.address}:{self.port}/v1/models/{request.model_name}/versions/{request.model_version}",
```
[the remark this answers is not in the document]
Comment (mkuczyns11) [reply client/python/lib/ovmsclient/tfs_compat/http/serving_client.py:118]: So you are suggesting that HttpClient should have requests.Session() object which would call .get method?

[the remark this answers is not in the document]
Comment (mkuczyns11) [reply client/python/lib/ovmsclient/tfs_compat/http/serving_client.py:118]: And if that's the case then where should it be initialized? We could create a session in _build() and pass it as an argument to __init__().

[the remark this answers is not in the document]
Comment (mzegla) [reply client/python/lib/ovmsclient/tfs_compat/http/serving_client.py]: Yeah, we could do that

**Review by michalkulakowski — APPROVED** [review]

**Pull request merged**